vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml

```diff
@@ -0,0 +1,4 @@
+type: io.gapi.vgen.ConfigProto
```

Comment:
This rename is part of Thomas' change, so if you submit before he does, then he would want to add the rename into his own PR; if you submit after, then you would want to include the rename.